### ISSUE TITLE
Extraction Error

### ISSUE THREAD

**Issue by StrollStars** [activescott/lessmsi#182]
**Describe the bug**
Error occurred during the extraction process.

**To Reproduce**
1. Use [7-Zip](https://www.7-zip.org/) to extract [FileMaker Pro.exe](https://we.tl/t-U40NtZ0uTt) and obtain the `FileMaker Pro 20.3.2` directory.
2. Use lessmsi to extract `\FileMaker Pro 20.3.2\Files\FileMaker Pro 20.msi`.
3. Wait for a while.

**Expected behavior**
Can be successfully extracted.

**Screenshots**
![image](https://github.com/activescott/lessmsi/assets/10942225/2e063b52-7865-46b4-b72e-88d9f6285ac7)

**Desktop (please complete the following information):**
 - OS: Windows 7 64bit
 - Browser Chrome
 - Version 1.10

**Smartphone (please complete the following information):**

**Additional context**

**Comment by activescott** [comment]
Thanks for reporting. I'm swamped with a project right now but I'll take a look at this in the coming couple weeks. Double check that the paths you chose to output to are valid as this _could_ be a factor here. Also worth creating a test file using that path yourself to make sure it's valid for your OS as configured. Feel free to tag me here if I get caught up and you're eager! :)

**Comment by StrollStars** [comment]
@activescott Thank you for the response, I'm not in a hurry. I changed the output path, but the issue persists. If the path is invalid, it should have thrown an error from the beginning. The error occurred after the progress bar had already reached more than halfway.

**Comment by activescott** [comment]
@StrollStars I tried to investigate this today but the filemaker file you linked isn't available anymore. I downloaded 20.3.2 from https://www.claris.com/resources/downloads/#win but when I look at it in 7-Zip I don't see the same files you saw (I only see an .msp as shown in attached screenshot). Can you either attach the installer you're using directly to this issue (you can usually drag & drop files here) or help me understand if I'm missing something in the steps.
<img width="1340" alt="Screenshot 2024-04-08 at 19 03 26" src="https://github.com/activescott/lessmsi/assets/213716/9b0acf27-41d7-489a-a54c-70c361a2cb5f">

**Comment by StrollStars** [comment]
@activescott I re-uploaded the [FileMaker Pro.exe](https://we.tl/t-LC3dOikvyy)，and the problem still exists in the [v1.11.0](https://github.com/activescott/lessmsi/releases/tag/v1.11.0).

**Comment by 0x0000beef** [comment]
It's all about UTF-8 filenames or transcoding, something like that. For example, two files from Ableton Live always crash the application. In particular, the Live installer uses internal 1252 codepage.
![By1mkZb 1](https://github.com/activescott/lessmsi/assets/40094560/e0efbaa6-339b-45b3-b3d1-fac0807fc3a2)

**Comment by StrollStars** [comment]
@0x0000beef Is there any solution? If it cannot be completely solved? Can it be skipped?

**Comment by 0x0000beef** [comment]
@StrollStars Nope, I just memorize where it gets stuck, find that spot in the list, select from the next item to the end and retrieve again, lol. In the case of Live, I only have to do it twice.

**Comment by activescott** [comment]
@StrollStars I tried to download that file you linked to but it doesn't work and says _Transfer expired
Sorry, this transfer has expired and is not available any more_. Do you have the same issue with the official FileMaker Pro installer at https://www.claris.com/resources/downloads/#win ?

If the installer at claris.com still gives you problems, it means it's something specific to the OS. Maybe the language installed from or the language selected for the current user? Or something else? 

I am using English Windows 11 VirtualBox VM that Microsoft provides for developers at https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/

**Comment by activescott** [comment]
@StrollStars Please try to upload that installer to https://www.dropbox.com/request/AcrMXznlJixuXmLBlrk0 instead of this expiring service

**Comment by StrollStars** [comment]
@activescott I have uploaded it, please check it.

**Comment by activescott** [comment]
Got it! I'm into a string of busy days but I'll get back to this. Don't hesitate to bug me though if you want an update :)
